### PR TITLE
run script and examples_lib

### PR DESCRIPTION
**Pull request by dsctt**
First pass. Looking to get any thoughts on the direction so far so we're on the same page. There is certainly more param handling (e.g. `n_steps`) to do and more content to add (e.g. algos).

Logs come out like this right now:
```
T:
 [[0.  1.  0.  0. ]
 [0.  0.  1.  0. ]
 [0.  0.5 0.  0.5]
 [0.  0.  0.  0. ]]
R:
 [[0 0 0 0]
 [0 0 0 0]
 [0 0 0 1]
 [0 0 0 0]]
gamma: 0.5
p0:
 [1. 0. 0. 0.]
phi:
 [[1 0 0]
 [0 1 0]
 [0 1 0]
 [0 0 1]]
Pi_phi:
 [array([0, 0, 0, 0])]

===== MDP =====

mc_states: all
v: [0.16405115 0.3032951  0.60659021 0.        ]
pi: [0 0 0 0]

===== AMDP =====

mc_states: all
v: [0.14451767 0.43563225 0.        ]
pi: [0 0 0 0]

mc_states: first
v: [0.14849499 0.29698997 0.        ]
pi: [0 0 0 0]
```

### CODE REVIEW

File: grl/run.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        raise NotImplementedError(f'{args.spec} not defined in examples_lib')
+
+    logging.basicConfig(format='%(message)s', level=logging.INFO)
+    if args.log:
+        pathlib.Path('logs').mkdir(exist_ok=True)
+        rootLogger = logging.getLogger()
+        rootLogger.addHandler(logging.FileHandler(f'logs/{args.spec}-{time.time()}.log'))
+
+    # Get (PO)MDP definition
+    T, R, gamma, p0, phi, Pi_phi = getattr(examples_lib, args.spec)()
```
Comment (camall3n) [inline grl/run.py:60]: Can we make this an API function? Like `examples_lib.load(something)` or `examples_lib.retrieve(something)`

**Review by camall3n — COMMENTED** [review]
This looks like a nice upgrade over what I was doing before, so I'm inclined to accept as-is

I'd love there to be an API function for loading/retrieving an example, rather than `getattr(examples_lib, args.spec)`

File: grl/run.py (hunk 1 above)
Comment (dsctt) [reply grl/run.py]: Good idea, done

**Review by camall3n — APPROVED** [review]
Looks great!

**Pull request merged**